### ISSUE TITLE
[UX] Appearance page: Denote the theme selected as admin theme (same as we do for the default theme).

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#1105]
![backdrop-appearance_page-default_theme_denoted_as_such](https://cloud.githubusercontent.com/assets/2423362/9292007/d720bc3e-4423-11e5-90cd-05c60767c394.png)

**Comment by BWPanda** [comment]
I like this idea. Here's a PR that achieves it: https://github.com/backdrop/backdrop/pull/3346

![image](https://user-images.githubusercontent.com/2385329/94989677-ea203500-05b9-11eb-8289-ac6648b08bd3.png)

Not only does this PR add '(admin theme)' to the end of the name of the admin theme, but it also makes it display second in the list after the default theme (i.e. first theme is always the default, second is always admin, rest are other enabled themes in alphabetical order).

I didn't think it necessary to make the name of the admin theme bold (it's not as important as the default theme), but that's easy enough to change.

**Comment by stpaultim** [comment]
Adding a screenshot of the entire appearance page to see better what is currently happening. 

We do currently indicate which theme is being used as admin theme, but it's at the bottom of the page and I would guess that most new users do not notice this. 

I like the idea behind the PR. My concern is that simply adding the words "admin theme" might not be enough. If I were a new user, I might see these words and think that it simply means that "Seven" is an admin theme - and I might not understand that it is also the currently active admin theme. 

Should the words in parenthesis be "Selected admin theme" or "default admin theme" or "active admin theme." Something to indicate that it THE current admin theme and not simply any admin theme. 


CURRENT SITUATION - BEFORE PR:

![Screen_Capture_Result](https://user-images.githubusercontent.com/3144571/94999546-a27cb780-057f-11eb-93f8-31aac5ac6395.png)

**Comment by stpaultim** [comment]
Here is a screenshot of what happens when Basis is selected as both admin theme and default theme:

![Screen_Capture_Result](https://user-images.githubusercontent.com/3144571/94999675-a3621900-0580-11eb-83b7-a0560693f774.png)

**Comment by bugfolder** [comment]
I think we should be consistent within the page at what we call the admin theme: is it "admin theme" (used at the top of the Appearance page" or "Administration theme" (used at the bottom of the page)?

I note that in the Layouts listing, the corresponding layouts are called "Default Layout" and "Default Administrative Layout", which is yet another choice of wording (and capitalization).

(FWIW, I rather like @stpaultim's "active public theme" and "active admin theme", but whatever we use, it should be consistent within the page.)

**Comment by bugfolder** [comment]
With thanks to @BWPanda for the original PR, I've created a new PR with the change of calling it "administration theme" at the top of the page, to match what's on the bottom of the page. Otherwise, it's the same as BWPanda's, and I think is ready for review and testing.

**Comment by bugfolder** [comment]
Coding standard failure is for lines unrelated to this PR. The functional failure is the usual bogus testUninstallLocale failure (also unrelated to this PR).

**Comment by docwilmot** [comment]
I'm not sure about this though. To me it seems likely to confuse users since it puts admin themes at the same level as the front end themes when they are really intended to be different things. 

Can we instead consider removing the admin theme icon from the list and putting it in the admin theme fieldset with the select list?

**Comment by klonos** [comment]
Thanks @bugfolder 🙏🏼 ...this change brings us in par with the Drupal UI:
![image](https://github.com/backdrop/backdrop-issues/assets/2423362/70627194-722f-4be0-943f-113855e4bdeb)

Other than a code suggestion I left in the PR, this is WFM/RTBC 👍🏼 

@docwilmot if we were to change the UI, I would prefer it if we switched to a table. I was thinking basically something similar to the modules page (consistency), but with the thumbnail previews shown as well. Kinda like the table with the layout templates in `admin/structure/layouts/settings`:
![image](https://github.com/backdrop/backdrop-issues/assets/2423362/2257c448-ba71-4909-8dec-2fab469c9a2f)

When it comes to selecting something to be the default, I like the UI pattern in the language configuration page at `admin/config/regional/language`:
![image](https://github.com/backdrop/backdrop-issues/assets/2423362/816bbd36-b3d1-4bab-b9e5-ef827cd06967)

So basically something like a crossbreed of all these table-based forms/UIs. I'll see if I can whip up a mockup and file a separate issue for this, but the table columns would basically be:
- enable/disable checkboxes
- theme thumbnail and name
- a "default theme" column of radios
- an "admin theme" column of radios

**Comment by klonos** [comment]
...also, this is a small UX task that can go in with the next bug fix release, so setting the issue labels accordingly.

**Comment by bugfolder** [comment]
Thanks, @klonos, for the review; your code suggested changes made. Tests are passing (except CSpell, for things not in this PR, and the usual `testLayoutUpgrade` gremlin, also unrelated). Can you (hopefully final) review?

**Comment by stpaultim** [comment]
I'm going to mark this WFM. This is a simple fix to make the existing UI for themes a little better. 

I agree that we could do more to improve how admin themes are handled on this page, but I think that is a different issue.

**Comment by quicksketch** [comment]
I merged https://github.com/backdrop/backdrop/pull/4522 into 1.x and 1.26.x. I also agree with @docwilmot and @stpaultim... we could do more to improve this page and specifically how admin themes are represented. But looking at the [before](https://github.com/backdrop/backdrop-issues/issues/1105#issuecomment-703149509) and [after](https://github.com/backdrop/backdrop-issues/issues/1105#issuecomment-1805193081), this is a easy incremental improvement.
